schema.json

```
@@ -919,6 +928,26 @@
           "type": "string",
           "format": "uri"
         },
+        "resellerURLs": {
+          "type": "array",
+          "items": {
+              "type": "object",
+              "properties": {
+                "name": {
+                    "type": "string",
+                    "minLength": 1
+                },
+                "region": {
+                    "type": "string",
+                    "minLength": 1
+                },
```

Needs to be an array of enum

Let's go with the G20 excluding individual EU members. We can always expand later;

```suggestion
                "regions": {
                    "type": "array",
                    "items": {
                    	"enum": [
                    		"Argentina",
                    		"Australia",
                    		"Brazil",
                    		"Canada",
                    		"China",
                    		"European Union",
                    		"India",
                    		"Indonesia",
                    		"Japan",
                    		"Mexico",
                    		"Russia",
                    		"Saudi Arabia",
                    		"South Africa",
                    		"South Korea",
                    		"Turkey",
                    		"United States",
                    		"Other
                    	]
                    }
                },
```